lokinet/lokinet-exit.dockerfile

```
@@ -5,17 +5,13 @@ COPY contrib/lokinet-exit.ini /var/lib/lokinet/conf.d/exit.ini
 
 # set up system configs
 COPY contrib/lokinet-exit-sysctl.conf /etc/sysctl.d/00-lokinet-exit.conf
-COPY contrib/lokinet-exit-rc.local.sh /etc/rc.local
-RUN /bin/bash -c 'chmod 700 /etc/rc.local'
+COPY --chmod=700 contrib/lokinet-exit-rc.local.sh /etc/rc.local
 
-COPY contrib/print-lokinet-address.sh /usr/local/bin/print-lokinet-address.sh
-RUN /bin/bash -c 'chmod 700 /usr/local/bin/print-lokinet-address.sh'
+COPY --chmod=700 contrib/print-lokinet-address.sh /usr/local/bin/print-lokinet-address.sh
 
 # setup cron jobs
-COPY contrib/lokinet-kill-scans.sh /usr/local/bin/lokinet-kill-scans.sh
-RUN /bin/bash -c 'chmod 700 /usr/local/bin/lokinet-kill-scans.sh'
-COPY contrib/lokinet-update-exit-address.sh /usr/local/bin/lokinet-update-exit-address.sh
-RUN /bin/bash -c 'chmod 700 /usr/local/bin/lokinet-update-exit-address.sh'
+COPY --chmod=700 contrib/lokinet-kill-scans.sh /usr/local/bin/lokinet-kill-scans.sh
+COPY --chmod=700 contrib/lokinet-update-exit-address.sh /usr/local/bin/lokinet-update-exit-address.sh
```

These ones *are* privileged -- and thus properly belong in /usr/local/sbin rather than `bin`

```suggestion
COPY --chmod=700 contrib/lokinet-kill-scans.sh /usr/local/sbin/lokinet-kill-scans.sh
COPY --chmod=700 contrib/lokinet-update-exit-address.sh /usr/local/sbin/lokinet-update-exit-address.sh
```